firstvoices/backend/migrations/0020_rename_notes_character_note.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 4.2.1 on 2023-05-15 19:19
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        (
+            "backend",
+            "0019_alter_category_options_alter_dictionaryentry_options_and_more",
+        ),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name="character",
+            old_name="notes",
+            new_name="note",
```

Comment:
It looks like there are a few string references to this model name which didn't get changed (on a few admin panel fields https://github.com/First-Peoples-Cultural-Council/fv-be/blob/503d3e2fbf64e4e0080b32748e11210ba31a8682/firstvoices/backend/admin/characters_admin.py#L42 and https://github.com/First-Peoples-Cultural-Council/fv-be/blob/503d3e2fbf64e4e0080b32748e11210ba31a8682/firstvoices/backend/admin/characters_admin.py#L86

Reply:
Good catch, thank you!